src/pages/StatusPage.jsx

```diff
@@ -0,0 +1,143 @@
+    console.log(err)
+    setState((prevState) => ({...prevState, [country]: "ERROR"}))
+    })
+}
+
+  useEffect(() => {
+    const baselinePolicyId = { uk: 1, us: 2, ca: 3, ng: 4};
+
+    for (let country of COUNTRY_CODES) {
+      getAPIRoute(country, 
+                  `/policy/${baselinePolicyId[country]}`, 
+                  setCountryStatuses, 
+                  countryApiCall)
+      getAPIRoute(country, "/metadata", setMetaDataStatuses, countryApiCall)
+      getAPIRoute(country, 
+                  `/${country}/calculate`, 
+                  setCalculateStatuses, 
+                  apiCall, 
+                  {   
+                      household: DEFAULT_COUNTRY_HOUSEHOULD_INPUTS[country], 
+                      policyId: `${baselinePolicyId[country]}`
+                  })
+    }
+  }, [])
+
+  return (
+    <>
+      <Header />
+        <div
+          style={{
+            paddingLeft: 50,
+            paddingRight: 50,
+            paddingTop: 70,
+            display: "flex",
+            flexDirection: "column",
+            alignItems: !mobile && "center",
+            marginBottom: 50,
+            textAlign: "center"
+          }}
+        >
+          <div
+              style={{
+                paddingRight: 0,
+                paddingLeft: 0,
+              }}
+            >
+            <h1>{`PolicyEngine's API Status Information`}</h1>
+            <section>
+              <h3>Country Pages</h3>
+              <h6>{`These requests represent accessing the home page for PolicyEngine's supported countries.`}</h6>
+                {Object.keys(countryStatuses).map((countryStatus, idx) => 
+                    <div 
+                      key={`${idx + 1}-country`} 
+                      style={{ 
+                        color: STATUS_TEXT_COLORS[countryStatuses[countryStatus]["status"]],
+                        backgroundColor: STATUS_COLORS[countryStatuses[countryStatus]["status"]],
+                        maxWidth: "50%",
+                        marginTop: 2,
+                        marginLeft: "auto",
+                        marginRight: "auto"
+                      }} 
+                    >
+                        {countryStatus.toUpperCase()} - {countryStatuses[countryStatus]["status"] ? countryStatuses[countryStatus]["status"] : "Checking Status"} - {countryStatuses[countryStatus]["latency"] ? countryStatuses[countryStatus]["latency"] : "Calculating Latency"}
+                    </div> 
+                )}
+            </section>
+            <section>
+              <h3 style={{marginTop: 5}}>Metadata</h3>
+              <h6>{`These requests represent accessing the metadata for PolicyEngine's supported countries.`}</h6>
+              {Object.keys(metaDataStatuses).map((metaDataStatus, idx) => 
+                  <div 
+                    key={`${idx + 1}-metaData`} 
+                    style={{ 
+                        color: STATUS_TEXT_COLORS[metaDataStatuses[metaDataStatus]["status"]],
```

Comment:
Nice-to-have: could we wrap these in a functional component given the duplication?